### ISSUE TITLE
MacOS Not Updating

### ISSUE THREAD

**Issue by knitterb** [jublonet/bingwallpaper#3]
I'm finding that my image isn't always updated.  I think this is because the file changes (as it should) but the filename doesn't change and OS is caching the image since the filename didn't change.  Any chance the file could be downloaded and then renamed and then have the desktop background changed?

You can see that if you delete the background image that the background indeed doesn't change either, indicating the image much be cached elsewhere.

TL;DR MacOS doesn't check for a file chance, only a filename change.

**Comment by mynetx** [comment]
I'll take a look. Thanks for telling! :)

**Comment by mynetx** [comment]
Hi Brandon,
Can you please try this version and tell me if the issue is fixed and the wallpaper updates correctly?

[BingWallpaper-macOS.app.zip](https://github.com/jublonet/bingwallpaper/files/721938/BingWallpaper-macOS.app.zip)
